CMakeLists.txt

```diff
@@ -118,6 +118,19 @@ else()
   message(STATUS "Detected ROOT_VERSION ${ROOT_VERSION}")
 endif()
 
+#---Try to download a dummy (non-existing) file to check internet connection--------------------
+file(DOWNLOAD https://root.cern/files/dummy.txt ${CMAKE_CURRENT_BINARY_DIR}/dummy.txt
+  TIMEOUT 100 STATUS DOWNLOAD_STATUS
```

Comment:
Is that 100 seconds? I think 10 is enough :-) People might get seriously worried if their CMake just sits there for two minutes. It might also be nice to show
```
message(STATUS "Checking internet connectivity...")
...
if(...)
  message(STATUS "Yes")
else()
  message(STATUS "No: will not automatically download external dependencies")
endif()
```

Reply:
@Axel-Naumann :wow: you're right, sorry, I would say that even 3 seconds would be enough... And I will add the messages, thanks for the suggestion